devices/quotra.js

```diff
@@ -0,0 +1,23 @@
+const exposes = require('../lib/exposes');
+const fz = {...require('../converters/fromZigbee'), legacy: require('../lib/legacy').fromZigbee};
+const tz = require('../converters/toZigbee');
+const reporting = require('../lib/reporting');
+const extend = require('../lib/extend');
+const e = exposes.presets;
+
+module.exports = [
+    {
+        zigbeeModel: ['Dimmer_us'],
+        model: 'Dimmer_us',
```

Comment:
The manufacturer is defunct and the product was only ever sold on amazon so current information is hard to come by.

Reply:
The manufacturer is defunct and the products were to my knowledge only ever sold on amazon so current information is hard to come by.

The dimmer came in a plain brown box with no markings. The amazon.com.au page is still up for the dimmer - https://www.amazon.com.au/dp/B07CVL9SZF and is the only reference I can find that is still alive. The amazon listed part number of B07CVL9SZF is the same as the stock number so it is unlikely to be valid as googling it just returns pages with links to the Amazon listing

For the light strip, there is again an amazon.com.au listing still alive https://www.amazon.com.au/dp/B07JHL6DRV and again no proper model number just the internal ID

Reply:
So lets use those numbers, it's better than nothing.

Reply:
Works for me. Just for my future reference as it is another item you changed in the merge. 
If adding another device the vendor: 'xxxx', part doesn't have to exactly match what the device returns in order to be valid.  It can be sanitized for consistency?

Reply:
Vendor is purely cosmetical (for docs/frontend), matching is done based on the `zigbeeModel`.